### ISSUE TITLE
When focused, Notepad++ scroll behavior doesn't match system scroll behavior

### ISSUE THREAD

**Issue by iamtheddrman** [notepad-plus-plus/notepad-plus-plus#2885]
### Description of the Issue
I use Evoluent Mouse Manager to enable accelerated scrolling on my scroll wheel. If I quickly scroll the wheel, it scrolls more lines than the default 3. For all programs I use on a regular basis except Notepad++, this behavior is honored. Notepad++ honors this behavior when it is NOT the active window, but forces the strict 3-line scroll behavior when focused. Furthermore, the 3-line scroll behavior is forced upon other inactive windows when Notepad++ is focused.

Note that this has been tested with multiple document types (plain text, XML, JS, Batch) with all exhibiting the same behavior.

### Steps to Reproduce the Issue
STR 1:
1. Enable accelerated scrolling through the mouse driver of your choice (I used Evoluent Mouse Manager)
2. Open a long document in Notepad++
3. Scroll quickly on the mouse wheel

STR 2:
1. Enable accelerated scrolling through the mouse driver of your choice (I used Evoluent Mouse Manager)
2. Enable inactive window scrolling (CP in Win10, or via MouseWiz on older versions of Windows)
3. Open a long document in Notepad++
4. Scroll quickly on the mouse wheel with the pointer inside Notepad++
5. Scroll quickly on the mouse wheel with the pointer in another application
5. Make a different application active
6. Scroll quickly on the mouse wheel with the pointer inside Notepad++
7. Scroll quickly on the mouse wheel with the pointer in another application

### Expected Behavior
STR 1: I would expect Notepad++ to use the accelerated scrolling that other programs honor (tested with Chrome, Chrome DevTools, MS Office 2011/2016 suite, WebStorm, VisualStudio 2013, and more).

STR 2: I would expect that the behavior would be consistent and honor accelerated scrolling settings.

### Actual Behavior
STR 1: Notepad++ is limited to 3-line scrolling, regardless of the speed at which I scroll.

STR 2: At steps 4/5, the scroll functionality is limited to the default 3-line scroll for both Notepad++ and all other applications on the system. At steps 6/7, the scroll functionality behaves with the accelerated functionality in both Notepad++ and all other applications on the system. It is disorienting to have my scroll functionality change based on which window is focused, as I switch between Notepad++ and other applications frequently throughout the day.

### Debug Information
Notepad++ v7.2.2   (32-bit)
Build time : Nov 27 2016 - 16:31:07
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : ON
Local Conf mode : OFF
OS : Windows 10
Plugins : ClipboardHelper.dll JSMinNPP.dll mimeTools.dll NppConverter.dll NppExport.dll NPPJSONViewer.dll PluginManager.dll XMLTools.dll

**Comment by dibyenshu** [comment]
Similar problem. I have smooth scrolling system-wide but with notepad++ is jerky/jumpy and hence I can't follow the text with my eyes while scrolling. This bug fix will be highly valued and it has forced me to switch to sublime. I would love to come back to notpad++ as soon as this bug is fixed.

**Comment by jaw4ik** [comment]
@dibyenshu +1. The same reason was moved to Sublime. Waiting for fix.

**Comment by MetaChuh** [comment]
please remove Evoluent Mouse Manager.

if your issue still persists, and it is still of relevance to you, please feel free to file a new issue as detailed and reproducible as possible.

thank you for your help and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)

**Comment by iamtheddrman** [comment]
Huh? I'm not going to remove my mouse manager that works with literally every other program just because it doesn't work with Notepad++! What a ridiculous "fix". As I stated, it works with dozens of other programs and editors, but doesn't with Notepad++. It's not an issue with the driver. It's an issue with Notepad++

**Comment by garyha** [comment]
Solved for me. I love Notepad++ by the way. That was driving me nutz, yet, you'll be happy to know it wasn't just them, I noticed the problem also in regedit and other programs.
New HP by0062st with Elan touchpad.
This older driver has many more settings. Essential I set Scroll Speed to 1 tick from Slow and unchecked Inertial Scroll.
Downgraded Elan driver from 21.2.17.1 to 18.2.19.1 Rev.A from ...
https://support.hp.com/us-en/drivers/selfservice/swdetails/hp-17-by0000-laptop-pc/20395856/model/21925617/swItemId/ob-209371-1 (or version ELAN 18.2.19.1_X64_WHQL according to the registry after install).
![image](https://user-images.githubusercontent.com/1115000/57437191-1c2b3d00-71f6-11e9-8f9f-6b723d83549e.png)

Those are just 2 changes that happened in the registry, which I determined with a diff of exports before and after:
```
[HKEY_CURRENT_USER\Software\Elantech\SmartPad]
"SC_InertialScroll_Enable"=dword:00000000
"SC_Speed"=dword:00000001
```
_I'm not convinced I have the best settings yet and real interested in whether anyone else finds something better. There's still a difference between various applications._
